samples/bluetooth/broadcast_audio_sink/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,12 @@
+# For LC3 the following configs are needed
+CONFIG_FPU=y
+CONFIG_LIBLC3=y
+# The LC3 codec uses a large amount of stack. This app runs the codec in the work-queue, hence
+# inctease stack size for that thread.
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+# LC3 lib requires floating point support in the c-lib NEWLIB is one way of getting that.
+CONFIG_NEWLIB_LIBC=y
+CONFIG_BT_HCI_ACL_FLOW_CONTROL=n
+CONFIG_BT_HCI_ACL_FLOW_CONTROL=n
+CONFIG_BT_HCI_VS_EXT=n
+CONFIG_BT_TINYCRYPT_ECC=y
```

Comment:
There is a duplicate.  Also, I think we should perhaps follow the pattern of not adding controller specific host configs here.  Maybe @Thalley has an opinion?